backend/src/lang/dialog-flow.d.ts

```diff
@@ -0,0 +1,102 @@
+/*
+This file contains some type definitions of DialogFlow.
+It is not a complet set of types, but covers the most important ones.
+All types are taken from the offical documentation.
+*/
+
+// Request interfaces
+
+interface DetectIntentRequest {
+    queryParams?: object, // object(QueryParameters)
+    queryInput?: QueryInput,
+    inputAudio?: string
+}
+
+// Response interfaces
+
+interface DetectIntentResponse {
+    responseId: string,
+    queryResult: QueryResult,
+    webhookStatus: object // object(Status)
+}
+
+// Base interfaces
+
+interface QueryInput {
+    audioConfig?: InputAudioConfig,
+    text?: TextInput,
+    event?: EventInput
+}
+
+interface InputAudioConfig {
+  audioEncoding: any,// enum(AudioEncoding)
+  sampleRateHertz: number,
+  languageCode: string,
+  phraseHints?: [string]
+}
+
+interface TextInput {
+    text: string,
+    languageCode: string
+}
+
+interface EventInput {
+    name: string,
+    parameters?: object,
+    languageCode: string
+}
+
+interface QueryResult {
+    queryText: string,
+    languageCode: string,
+    speechRecognitionConfidence: number,
+    action: string,
+    parameters: object, // TODO not typed?
```

Comment:
You might want to work at this // TODO before merge it into develop. If not ingore this comment.